tests/foreman/cli/test_errata.py

```diff
@@ -1091,22 +1092,28 @@ def test_positive_list_filter_by_product_id_and_org_label(self):
             set([])
         )
 
-    @stubbed()
-    def test_positive_list_filter_by_product_name(self):
```

Comment:
@oshtaier the stub was not removed, but the test was renamed to negative, 
from test_positive_list_filter_by_product_name to test_negative_list_filter_by_product_name